pandas/io/tests/parser/test_parsers.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+import os
+import nose
+
+import pandas.util.testing as tm
+
+from pandas import read_csv, read_table
+from pandas.core.common import AbstractMethodError
+
+from .parser_common import ParserTests
+from .c_parser_only import CParserTests
+from .parse_dates import ParseDatesTests
+from .compression import CompressionTests
+from .multithread import MultithreadTests
+from .python_parser_only import PythonParserTests
+
+
+class BaseParser(CompressionTests, MultithreadTests,
+                 ParseDatesTests, ParserTests):
+    def read_csv(self, *args, **kwargs):
+        raise NotImplementedError
+
+    def read_table(self, *args, **kwargs):
+        raise NotImplementedError
+
+    def float_precision_choices(self):
+        raise AbstractMethodError(self)
+
+    def setUp(self):
+        import warnings
```

Comment:
remove the filter warnings, we don't filter anywhere (except very explicty). this is hiding warnings